cmd/execute.cpp

```diff
@@ -128,82 +69,70 @@ int main(int argc, char* argv[]) {
 
     CLI11_PARSE(app, argc, argv);
 
+    Logger::default_logger().set_local_timezone(true);  // for compatibility with TG logging
+
     // Check data.mdb exists in provided directory
     boost::filesystem::path db_file{boost::filesystem::path(db_path) / boost::filesystem::path("data.mdb")};
     if (!boost::filesystem::exists(db_file)) {
-        std::clog << "Can't find a valid TG data file in " << db_path << std::endl;
-        return -2;
+        SILKWORM_LOG(LogError) << "Can't find a valid TG data file in " << db_path << std::endl;
+        return -1;
     }
 
     // Check provided map size is valid
-    auto map_size{silkworm::parse_size(map_size_str)};
+    auto map_size{parse_size(map_size_str)};
     if (!map_size.has_value()) {
-        std::clog << "Invalid --lmdb.mapSize value provided : " << map_size_str << std::endl;
+        SILKWORM_LOG(LogError) << "Invalid --lmdb.mapSize value provided : " << map_size_str << std::endl;
         return -2;
     }
 
-    std::clog << "Starting block execution. DB: " << db_file << std::endl;
+    SILKWORM_LOG(LogInfo) << "Starting block execution. DB: " << db_file << std::endl;
 
-    silkworm::lmdb::DatabaseConfig db_config{db_path, *map_size};
+    lmdb::DatabaseConfig db_config{db_path, *map_size};
     db_config.set_readonly(false);
-    std::shared_ptr<silkworm::lmdb::Environment> env{nullptr};
-    std::unique_ptr<silkworm::lmdb::Transaction> txn{nullptr};
+    std::shared_ptr<lmdb::Environment> env{lmdb::get_env(db_config)};
```

Comment:
This may throw.
Without a try catch you may have program dump

Reply:
On macOS the system displays the uncaught exception. What happens on Windows?

Reply:
Same on Windows. A pop-up appears for un-catched exception.
But we both work on graphical UI with terminal open.
On a headless linux program dumps memory without detail

Reply:
Right. I've restored the try/catch to be on the safe side.